src/mlpack/tests/linear_svm_test.cpp

```diff
@@ -454,19 +454,22 @@ BOOST_AUTO_TEST_CASE(LinearSVMLBFGSSimpleTest)
 {
   const size_t numClasses = 2;
   const double lambda = 0.0001;
+  const double delta = 1.0;
 
-  // A very simple fake dataset
+  // A very simple fake dataset.
   arma::mat dataset = "2 0 0;"
                       "0 0 0;"
                       "0 2 1;"
                       "1 0 2;"
                       "0 1 0";
 
-  //  Corresponding labels
+  // Corresponding labels.
   arma::Row<size_t> labels = "1 0 1";
 
+  ens::L_BFGS opt;
   // Create a linear svm object using L-BFGS optimizer.
-  LinearSVM<arma::mat> lsvm(dataset, labels, numClasses, lambda);
+  LinearSVM<arma::mat> lsvm(dataset, labels, numClasses, lambda,
+    delta, false, opt);
```

Comment:
Oops, minor style issue---can you indent the wrapped line twice?  (This happens a few times)